SQL/tgstation_schema.sql

```diff
@@ -387,6 +387,17 @@ CREATE TABLE `round` (
 /*!40101 SET character_set_client = @saved_cs_client */;
 /*!40103 SET TIME_ZONE=@OLD_TIME_ZONE */;
 
+--
+-- Table structure for table `schema_revision`
+--
+DROP TABLE IF EXISTS `schema_revision`;
+CREATE TABLE `schema_revision` (
+  `major` int(10) unsigned NOT NULL,
+  `minor` int(10) unsigned NOT NULL,

```

Comment:
These could just be `TINYINT(3)`, I doubt we're going to pushing past 255 versions. Or even then, `SMALLINT(5)` for 65k limit.